src/utils/spotify.js

```diff
@@ -196,7 +199,7 @@ export const searchSpotify = (query, type = "track", limit = 20) => {
     limit: limit.toString(),
   });
 
```

Comment:
`https://accounts.spotify.com` is duplicated also 4 times. Please, create a constant: `SPOTIFY_ACCOUNT_URL`